src/getCoinCombination.test.js

```diff
@@ -7,4 +7,24 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test('The function should return an array', () => {
```

Comment:
```suggestion
  test('should return a dime, when cents is equal to 10 ', () => {
```
fix test names everywhere